### PR TITLE
Navbar and sidebar

### PR DESCRIPTION
**Pull request by sushil-mod**
### what to check :

- Navbar Jsx & css
- Sidebar jsx & css

## Screenshot 
vl

### CODE REVIEW

File: src/components/sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    width:3.5rem;
+    height:3.5rem;
+}
+/* hover styles */
+.item-icon:hover{
+    background-color: #666161;
+}
+.item-title:hover {
+    cursor: pointer;
+    /* border-radius: 10px; */
```
Comment (Kothalkarpriya) [inline src/components/sidebar/Sidebar.css:36]: If not needed you can remove the commented code

File: src/components/sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  return <>
+    
+    <div className='sidebar' >
+        <ul className='sidebarlist'>
+           { data.map( ({title , icon }) => 
+               <li  > 
+               <NavLink to='/' className='row flex-center item-title'  >             
+                    <div className='side-icon'>
+                        <div className='icon-div flex-center ' > {icon}</div>
+                    </div>
```
Comment (Kothalkarpriya) [inline src/components/sidebar/Sidebar.jsx:48]: This is a good idea that you are using different component for these icons at once

**Review by Kothalkarpriya — APPROVED** [review]
Good Start! You can merge the changes

File: src/components/sidebar/Sidebar.css (hunk 1 above)
Comment (sushil-mod) [reply src/components/sidebar/Sidebar.css:36]: sure .

File: src/components/navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.menu__link_title{
+    font-size: 2rem;
+}
+.menu__container{
+    margin-right: 1rem;
+    font-size: 1.5rem;
+}
+.menu__container_link{
+    border: 1px solid #3EA4FC ;
+    padding: 0.5rem;
```
Comment (munjalSaksham71) [inline src/components/navbar/Navbar.css:21]: Might be in px for small padding/margins.

File: src/components/navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { FaBars } from 'react-icons/fa';
+import { CgProfile } from 'react-icons/cg'
```
Comment (munjalSaksham71) [inline src/components/navbar/Navbar.jsx:4]: You could create a file from where you can export all the icons.

File: src/components/navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    </Link> 
+
+    </div>
+
+    <div className='menu__container gap-lg flex-space-btw '>
+
+                <div className='menu__container_link cur-pointer'>
+                    <Link className='login_link flex-center' to="/"  > <CgProfile /> Login  </Link>
+                </div> 
+    </div>
```
Comment (munjalSaksham71) [inline src/components/navbar/Navbar.jsx:27]: Please Format Document By using prettier or something else

File: src/components/sidebar/Sidebar.css (hunk 1 above)
Comment (munjalSaksham71) [inline src/components/sidebar/Sidebar.css:36]: If not needed- Remove Comment

**Review by munjalSaksham71 — APPROVED** [review]
Looks Good. Added Few Suggestion , Please Look.

**Pull request merged**